site/app/templates/polls/PollForm.twig

```diff
@@ -202,7 +222,20 @@
         }
         submitErrorChecks();
     };
+    window.onload = function() {
+        toggleTimerInputs();
+    };
+    function toggleTimerInputs() {
+        const enableTimerCheckbox = $("#enable-timer");
+        const timerInputs = $(".poll-timer");
 
+        if (enableTimerCheckbox.prop("checked")) {
+            $("#timer-inputs").show();
+        } else {
+            $("#timer-inputs").hide();
+        }
+    }
```

Comment:
It would be good to move this to `polls.js` so the JS code gets linted.

Reply:
Fixed